src/installer.ts

```diff
@@ -114,7 +114,48 @@ async function downloadRelease(version: string): Promise<string> {
   return await tc.cacheDir(extPath, "protoc", version);
 }
 
-function getFileName(version: string): string {
+/**
+ *
+ * @param osArch - A string identifying the operating system platform for which the Node.js binary was compiled.
+ * See https://nodejs.org/api/os.html#osplatform for possible values.
+ * @returns Suffix for the protoc filename.
+ */
+function fileNameSuffix(osArch: string): string {
+  switch (osArch) {
+    case "x64": {
+      return "x86_64";
+    }
+    case "arm64": {
+      return "aarch_64";
+    }
+    case "s390x": {
+      return "s390_64";
+    }
+    case "ppc64": {
+      return "ppcle_64";
+    }
+    default: {
+      return "x86_32";
+    }
+  }
+}
+
+/**
+ * Returns the filename of the protobuf compiler.
+ *
+ * @param version - The version to download
+ * @param osPlat - The operating system platform for which the Node.js binary was compiled.
+ * See https://nodejs.org/api/os.html#osplatform for more.
+ * @param osArch - The operating system CPU architecture for which the Node.js binary was compiled.
+ * See https://nodejs.org/api/os.html#osplatform for more.
```

Comment:
```suggestion
 * @param osArch - The operating system CPU architecture for which the Node.js binary was compiled.
 * See https://nodejs.org/api/os.html#osarch for more.
```


The description and reference URL were for the operating system platform/`os.platform()`, whereas this parameter is the architecture/`os.arch()`.